vendor/engines/sanger_sequencing/db/migrate/20250120141340_add_sample_primer_name.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class AddSamplePrimerName < ActiveRecord::Migration[4.2]
```

Comment:
Is there any reason to use `Migration[4.2]`? I see we have a bunch of different versions. For instance, I see we used 7.0 for `AddProductStartTimeDisabled` a month ago

Reply:
I was using the same version number as previous migrations (for this engine). This might be the reason why if_no_exists parameter didn't work and I had to manually check the column existance